source/extensions/filters/network/rocketmq_proxy/active_message.cc

```diff
@@ -0,0 +1,299 @@
+#include "extensions/filters/network/rocketmq_proxy/active_message.h"
+
+#include "extensions/filters/network/rocketmq_proxy/conn_manager.h"
+#include "extensions/filters/network/rocketmq_proxy/topic_route.h"
+
+#include "rapidjson/stringbuffer.h"
```

Comment:
Envoy is now actively trying to get rid of its depends on rapidjson #4705 we should convert to JSON using protobuf

Reply:
Now replaced with Protobuf.